### PR TITLE
chore(pd): sync get_lr from pt to pd

### PR DESCRIPTION
**Pull request by njzjz**
## Summary by CodeRabbit

* **Refactor**
  * Updated learning rate scheduling implementation with enhanced type annotations and more flexible parameter handling for improved code clarity.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: deepmd/pd/train/training.py [hunk 1]
```diff
@@ -30,6 +30,9 @@
 from deepmd.common import (
     symlink_prefix_files,
 )
+from deepmd.dpmodel.utils.learning_rate import (
```
[the remark this answers is not in the document]
Comment (njzjz) [reply deepmd/pd/train/training.py:33]: I don't see the meaning of reimporting it.
